lib/stdlib/doc/src/erl_tar.xml

```diff
@@ -248,7 +248,7 @@
       <name since="">create(Name, FileList) ->RetValue </name>
       <fsummary>Create a tar archive.</fsummary>
       <type>
-        <v>Name = filename()</v>
+        <v>Name = filename_all()</v>
```

Comment:
I think we'd prefer `file:filename_all()` here since it's not obvious what it refers to.

I also noticed that `open/2` has the wrong type too, can you update it?